ursina/physics.py

```diff
@@ -61,24 +61,25 @@ def __init__(self, mesh=None, center=(0,0,0)):
         self.center = center
 
 
-
-class PhysicsBody:
-    # copy the animation functionality from Entity
+class _PhysicsBody:
+    # Not worth rewriting from scratch

```

Comment:
This is not really why. I preferred the previous comment.

Reply:
I didn't leave it on purpose, I wrote it for myself x)